releasenotes/notes/trace-normalization-metric-fix-count-a9decee155c70926.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fix endlessly growing `traces_dropped` and `spans_malformed` metric counts.
```

Comment:
Prefix this with “APM: “ and maybe phrase it better as “APM: Fixed a provlem where ... and ... where showing incorrect metrics.”

Reply:
Updated